tests/Console/EloquentCommandTest.php

```diff
@@ -45,14 +45,25 @@ public function testCommand()
         $this->assertMatchesTxtSnapshot($actualContent);
 
         $display = $tester->getDisplay();
-        $this->assertRegExp(
-            ';Unexpected no document on Illuminate\\\Database\\\Eloquent\\\Model;',
-            $display
-        );
-        $this->assertRegExp(
-            ';Wrote expected docblock to .*/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php;',
-            $display
-        );
+        if(is_callable([$this, 'assertMatchesRegularExpression'])) {
+            $this->assertMatchesRegularExpression(
+                ';Unexpected no document on Illuminate\\\Database\\\Eloquent\\\Model;',
+                $display
+            );
+            $this->assertMatchesRegularExpression(
+                ';Wrote expected docblock to .*/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php;',
+                $display
+            );
+        } else {
+            $this->assertRegExp(
+                ';Unexpected no document on Illuminate\\\Database\\\Eloquent\\\Model;',
+                $display
+            );
+            $this->assertRegExp(
+                ';Wrote expected docblock to .*/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php;',
+                $display
+            );
+        }
```

Comment:
Please revert these changes; assertRegExp works fine; the warning may be annoying but IMHO no need for duping the code here.